clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -0,0 +1,233 @@
+import { CONSENT_COOKIE_NAME, ConsentBannerOptions, CookieKeyConsent } from "fides-js";
+
+// The fides-consent-demo.html page is wired up to inject the
+// `fidesConsentBannerOptions` into the Fides.banner(...) function
+declare global {
+    interface Window {
+        fidesConsentBannerOptions?: ConsentBannerOptions;
+    }
+}
+
+// We define a custom Cypress command
+declare global {
+    namespace Cypress {
+        interface Chainable {
+            visitConsentDemo(bannerOptions?: ConsentBannerOptions): Chainable<any>;
+        }
+    }
+}
+
+
+describe("Consent banner", () => {
```

Comment:
Hah, awesome! I'm glad these tests survived the wash.

What do you think about moving this to fides-js/cypress instead? I feel like it might make more sense to have the tests that are very focused on fides-js over there.

Reply:
ah yeah that makes more sense to me

Reply:
Let's do this as a follow-up ticket though, get this all nice and stable first.

Reply:
ticketed and documented in description of this PR- https://github.com/ethyca/fides/pull/3191#issue-1689319530